### ISSUE TITLE
Deprecated API usage

### ISSUE THREAD

**Issue by dimonpvt1236** [klaviyo/magento2-klaviyo#316]
### Description

In klaviyo  developer tools in section API revisions showed deprecated APi usage endpoints, it's endpoint used from this plugin. Can you update it?
![api deprecated](https://github.com/user-attachments/assets/08a16da2-8874-4ef0-a95e-4829bb97f9f2)

**Comment by cykolln** [comment]
Hi @dimonpvt1236 thanks for raising. We are working on getting all extension revisions up to date and guarantee that they will be ready to go well before the retirement date of 2025-08-15.

**Comment by davidtabat** [comment]
Any updates on the.issue so far?

**Comment by davidtabat** [comment]
> Due to the fact that a normal change in some projects like us may take 2-3 weeks to implement, I am hoping this will be updated at least a month before.
Agree! If not than I would say Klaviyo simply will have to delay retirement of old API.

**Comment by smoucka** [comment]
Hi, all. We are targeting an extension version release at the end of April. The release will incorporate the latest Klaviyo API revisions. Thanks for your patience.

**Comment by cykolln** [comment]
Hi folks - thanks for the patience here. We have the changes ready (#324) and are targeting release for the new extension version on April 16th once the 4/15/2025 revisions are in GA.

**Comment by sprankhub** [comment]
@cykolln, any news? 😁 Could you give an update here once the release is published?
